.github/workflows/01-code-review-approved.yaml

```diff
@@ -0,0 +1,66 @@
+name: 01-code_review_approved
+on:
+  pull_request_review:
+    types: [submitted]
+    branches:
+      - 'master'
+      - 'main'
+    paths-ignore:
+      - '**.yaml'
+      - '**.md'
+      - '**.sh'
+      - 'website/*'
+      - 'examples/*'
+      - 'docs/*'
+      - 'build/*'
+      - '.github/*'
+
+env:
+  CUSTOMER: ${{ secrets.CUSTOMER }}
+  USERNAME: ${{ secrets.USERNAME }}
+  PASSWORD: ${{ secrets.PASSWORD }}
+  CLIENT_ID: ${{ secrets.CLIENT_ID_PROD }}
+  SECRET_KEY: ${{ secrets.SECRET_KEY_PROD }}
+  REGISTRY_USERNAME: ${{ secrets.REGISTRY_USERNAME }}
+  REGISTRY_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
+  GH_PERSONAL_ACCESS_TOKEN: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+
+concurrency:
+  group: code-review-approved
+  cancel-in-progress: true
+
+jobs:
+
+  binary-build:
+    if: ${{ github.event.review.state == 'approved' && 
+            contains( github.event.pull_request.labels.*.name, 'trigger-integration-test') && 
+            github.event.pull_request.base.ref == 'master' }} ## run only if labeled as "trigger-integration-test" and base branch is master
+    uses: ./.github/workflows/b-binary-build-and-e2e-tests.yaml
+    with:
+      COMPONENT_NAME: kubescape
+      CGO_ENABLED: 1
+      GO111MODULE: ""
+      GO_VERSION: "1.19"
+      RELEASE: ${{ github.ref_name}}
+      CLIENT: test
+    secrets: inherit
+
+
+  merge-to-master:
+    needs: binary-build
+    if: ${{ env.GH_PERSONAL_ACCESS_TOKEN && 
```

Comment:
[TODO]: We should have a step named `run-merge-to-master`, that step should store the output of the `if` statement so this step will look cleaner